### PR TITLE
remove-gen-codes-with-goapi

### PR DESCRIPTION
**Pull request by EvanLjp**
remove-gen-codes-with-goapi

### CODE REVIEW

**Review by kezhenxu94 — APPROVED** [review]

File: docs/en/setup/plugins/plugin-list.md [hunk 1]
```diff
@@ -8,9 +8,9 @@
 - Fetcher
 - Filter
 - Forwarder
+	- [meter-grpc-forwarder](./forwarder_meter-grpc-forwarder.md)
 	- [nativelog-grpc-forwarder](./forwarder_nativelog-grpc-forwarder.md)
 	- [nativelog-kafka-forwarder](./forwarder_nativelog-kafka-forwarder.md)
-	- [meter-grpc-forwarder](./forwarder_meter-grpc-forwarder.md)
```
Comment (wu-sheng) [inline docs/en/setup/plugins/plugin-list.md:13]: Why remove this?

Comment (EvanLjp) [reply docs/en/setup/plugins/plugin-list.md:13]: this is auto-generated, I would check the reason

**Comment by wu-sheng** [comment]
Have we removed submodule permanently?

File: docs/en/setup/plugins/plugin-list.md (hunk 1 above)
Comment (EvanLjp) [reply docs/en/setup/plugins/plugin-list.md:13]: only sequence change, order by letter

**Review by kezhenxu94 — APPROVED** [review]

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**